### ISSUE TITLE
[improvement] The docker image needs to be updated.

### ISSUE THREAD

**Issue by AshwinB-hat** [jnewmano/grpc-json-proxy#17]
I was having issues using the postman api's using the docker image. 

The error that was popping up was 
`Post "http://{{grpc server address}}/{{proto package}}/{{proto service}}": dial tcp: lookup {{grpc server address}}: no such host`

I thought it was a permissions issue. but turns out it doesn't exist in the latest release.

**Comment by jnewmano** [comment]
Anything in {{ … }} should be replaced with the values for the server you are trying to connect to. {{ grpc server }} should be a host name that is resolvable via DNS or an IP address.

**Comment by AshwinB-hat** [comment]
Yes, I did. just wanted to leave the details out in the issue comment.

Note: I have an M1 Mac with ARM architecture. And I get the below error on running 
`WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
` 
I should have led with this in the issue comment.
The proxy works fine with the latest release but the `docker run -it -p 7001:7001 vincenthcui/grpc-json-proxy` docker image does not.

**Comment by jnewmano** [comment]
That’s correct, I have not made a ARM based image available.
